### ISSUE TITLE
IDE: scope should have vertical axis labelling

### ISSUE THREAD

**Issue by giuliomoro** [BelaPlatform/Bela#120]
with value in units

incidentally, what about "units/div" as a label instead of "attenuation"?
and offset would be "offset(units)"

**Comment by LBDonovan** [comment]
The tricky bit here is what to do with the axis labels when one of the channels has a different value for y-offset and amplitude to the others. I guess the labels could simply ignore them...

I called it attenuation because currently the incoming signal is divided by that number. I'm not really sure why this is the case, maybe I will look into changing it to be units/div.

**Comment by giuliomoro** [comment]
You are right, different "units/div" per different channels are just going to make axis labelling a mess.

**Comment by LBDonovan** [comment]
The labels could do what the y-axis mouse-over indicator does now, which is simply ignore the per-channel unit/div?

**Comment by giuliomoro** [comment]
I guess it's the only choice available.
Ideally I'd want to be able to click on a line and get an actual x-y reading out of it (matlab-style)
